src/main/webapp/WEB-INF/pages/layout/layout-control.jsp

```diff
@@ -14,14 +14,14 @@
     <link href="<c:url value="/resources/css/main.css" />" rel="stylesheet" type="text/css">
 </head>
 <body>
-<div class="container">
+<div class="container container-padding">
```

Comment:
rename `container-padding`, `remove-left-padding` and `remove-right-padding` to something more meaningful.